app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    split_command = command.split()
+    comm, exist_file, path = split_command
+    path, new_file = os.path.split(path)
+
+    src_file = os.path.join(os.getcwd(), exist_file)
+
+    if not path:
+        return os.rename(exist_file, new_file)
```

Comment:
you shouldn't return the function value. Now it returns None -> that's ok. But what if tomorrow developers will change this to return some value? You should separate this into lines. 
Same for `return os.rename(dst_file1, dst_file2)`